napari/utils/events/containers/_selection.py

```diff
@@ -10,9 +10,16 @@
 
 
 class Selection(EventedSet[_T]):
-    """An unordered collection of selected elements, with a `current` item.
+    """An model of selected items, with a `active` and `current` item.
 
-    There can only be one 'current' item. There can be multiple selected items.
+    There can only be one `active` and one `current` item, but there can be
+    multiple selected items.  An "active" item is defined as a single selected
+    item (if multiple items are selected, there is no active item).  The
+    "current" item is mostly useful for (e.g.) keyboard actions: even with
+    multiple items selected, you may only have one current item, and keyboard
+    events (like up and down) can modify that current item.  It's possible to
+    have a current item without an active item, but an active item will almost
+    always be the current item.
```

Comment:
The "almost" is... curious. :joy: is this just hedging?

Reply:
heh... yeah I think it is 😂